loopchain/protos/loopchain.proto

```diff
@@ -50,6 +54,17 @@ message BlockSyncReply {
     required int32 unconfirmed_block_height = 6;
 }
 
+message HeightRequest {
+    optional string peer = 1;
+    optional string channel = 2; // channel ID for multichain network
+}
+
+message HeightResponse {
+    optional string peer = 1;
+    optional string channel = 2; // channel ID for multichain network
+    optional int32 height = 3;
+}
+
```

Comment:
Are they enough just to be optional ?

Reply:
They should be required params.

Thank you